cmd/trace-agent/subcommands/config/command.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package config implements 'trace-agent config' cli.
+package config
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/trace-agent/subcommands"
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/pkg/config/fetcher"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+	"go.uber.org/fx"
+
+	"github.com/spf13/cobra"
+)
+
+// Commands returns a slice of subcommands for the 'agent' command.
+func MakeCommand(globalParamsGetter func() *subcommands.GlobalParams) *cobra.Command {
+	return &cobra.Command{
+		Use:   "config",
+		Short: "Print the runtime configuration of a running trace-agent",
+		Long:  ``,
+		RunE: func(*cobra.Command, []string) error {
+			return configFct(globalParamsGetter())
+		},
+	}
+}
+
+func configFct(params *subcommands.GlobalParams) error {
+	return fxutil.OneShot(printConfig,
+		fx.Supply(config.NewAgentParams(params.ConfPath)),
+		config.Module(),
+	)
+}
+
+func printConfig(config config.Component) error {
+	fullConfig, err := fetcher.TraceAgentConfig(config)
+	if err != nil {
+		fmt.Printf("error fetching trace-agent configuration: %s", err)
+		return nil
```

Comment:
Can you explain why the error is not returned?
If an error is not needed, `printConfig` doesn't need to return anything.

Reply:
Because I forgot, thanks for catching it :+1: !